reconcile/utils/ocm/ocm.py

```diff
@@ -1673,6 +1676,9 @@ def get_cluster_addons(
             return results
 
         for item in items:
+            if required_state:
+                if item["state"] != required_state:
+                    continue
```

Comment:
Can we use `search` params on https://api.openshift.com/#/default/get_api_clusters_mgmt_v1_clusters__cluster_id__addons to filter by `state`? So the filtering logic is in server side, we don't need to do another post filter on client side.

Reply:
Getting an unrelated API error locally, while trying that. Will revisit later. Want to fix integration first.